assets/base.css

```diff
@@ -99,7 +99,7 @@ html.no-js .no-js-hidden {
   padding: 0 1.5rem;
 }
```

Comment:
It's the same behaviour we have with the footer section, no matter if you show social icons there or not, we always display them in the drawer. 
I feel it may make sense to work on this in the feature, but it should not be a blocker for this PR. I don't remember the context, but it's hard to decide what logic to follow when you have social-icons in three places the announcement bar, footer and drawer. And for example, if you want to show them in the footer but not in the announcement-bar, then should we show them in the drawer?